adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +105,30 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [BUAdSDKManager setCoppa:adConfiguration.childDirectedTreatment.integerValue];
+    self.bannerRenderer = [GADPangleRTBBannerRenderer new];
```

Comment:
Replace all other uses of 'new' with alloc init, the same applies to below.

i.e. `_bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];

Reply:
solved